kernel-util/src/main/java/org/sakaiproject/util/Validator.java

```diff
@@ -359,7 +359,27 @@ public static String limitFormattedText(String value, int length)
         return ret.toString();
     }
 
- 
+	/**
+	Checks whether the email address has got valid local domain
```

Comment:
What this method does is much more than this. You should explain it's checking if it's an allowed host part. Formatting of JavaDoc is wrong.